### DISCUSSION TITLE
How to get list of buffers including positions?

### DISCUSSION THREAD

**Discussion by tf42src** [LazyVim/LazyVim#3766] [Q&A]
I am able to retrieve buffers with `vim.api.nvim_list_bufs()`. Unfortunately the order does not reflect the current order (after moving buffers with `[B`). Executing `:buffers` also shows the "incorrect" order.

![Screenshot from 2024-06-22 16-38-25](https://github.com/LazyVim/LazyVim/assets/9712861/446db864-e94b-4fc5-9a9d-8aa83cb57045)

Related to my other question [here](https://github.com/LazyVim/LazyVim/discussions/3746).

**Comment by Old-Farmer** [comment]
[B use bufferline.nvim command, and no effect to neovim core buffer management. It only effects how the plugin manages and shows buffers.

**Reply by tf42src** [reply]
> [B use bufferline.nvim command, and no effect to neovim core buffer management. It only effects how the plugin manages and shows buffers.

Ah, that makes sense. Thanks!
